### DISCUSSION TITLE
Node JS Version No longer Showing

### DISCUSSION THREAD

**Discussion by walherb** [JanDeDobbeleer/oh-my-posh#4046] [Troubleshoot]
Why is Node JS Version no longer showing in Node Applications in oh-my-posh version 17.9.0 and how to solve it if possible
![oh-my-posh](https://github.com/JanDeDobbeleer/oh-my-posh/assets/5444471/f39a20e7-edee-4df9-a72e-a07b95f0eb53)

**Comment by JanDeDobbeleer** [comment]
@walherb can you show the output of `oh-my-posh debug --plain` in that folder?

**Reply by walherb** [reply]
The output of oh-my-posh debug --plain is as below
![oh-my-posh-2](https://github.com/JanDeDobbeleer/oh-my-posh/assets/5444471/4551ddb9-1b28-42e6-b328-b2b19ee3fea8)

**Reply by JanDeDobbeleer** [reply]
@walherb can you copy paste the entire logs? That's what I need to see what's going on 🙏🏻

**Reply by walherb** [reply]
Here are the logs
[DEBUG] 18:41:33.112 shell.go:Getenv:397 → NO DATA
[TRACE] 18:41:33.112 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 18:41:33.112 shell.go:Getenv:397 → C:\Users\Herbert\AppData\Local
[TRACE] 18:41:33.112 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 18:41:33.112 shell_windows.go:CachePath() - 524.8µs
[DEBUG] 18:41:33.114 shell.go:Getenv:397 → NO DATA
[TRACE] 18:41:33.114 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 18:41:33.114 shell.go:resolveConfigPath:325 → No config set, fallback to default config
[TRACE] 18:41:33.114 shell.go:resolveConfigPath() - 0s
[TRACE] 18:41:33.114 shell.go:Init() - 1.5664ms
[TRACE] 18:41:33.114 shell.go:Flags() - 0s
[DEBUG] 18:41:33.114 config.go:loadConfig:127 → no config file specified, using default
[TRACE] 18:41:33.114 config.go:loadConfig() - 0s
[TRACE] 18:41:33.114 shell.go:Flags() - 0s
[DEBUG] 18:41:33.114 shell.go:Getenv:397 → NO DATA
[TRACE] 18:41:33.114 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 18:41:33.114 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[TRACE] 18:41:33.114 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.114 shell_windows.go:WindowsRegistryKeyValue:206 → ColorizationColor(DWORD): 0xC40078D7
[DEBUG] 18:41:33.122 shell.go:Shell:644 → process name: pwsh.exe
[TRACE] 18:41:33.122 shell.go:Shell() - 8.0356ms
[DEBUG] 18:41:33.122 shell.go:Getenv:397 → 7.3.5
[TRACE] 18:41:33.122 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 18:41:33.122 debug.go:PrintDebug:23 → Segment: Title
[TRACE] 18:41:33.122 shell_windows.go:Root() - 0s
[TRACE] 18:41:33.122 shell.go:Shell() - 0s
[TRACE] 18:41:33.122 shell.go:ErrorCode() - 0s
[TRACE] 18:41:33.122 shell_windows.go:IsWsl() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.122 shell.go:Pwd:427 → D:\Repos\Projects\expense-tracker\backend
[TRACE] 18:41:33.122 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.122 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[TRACE] 18:41:33.122 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.122 shell.go:User:548 → Herbert
[TRACE] 18:41:33.122 shell.go:User() - 0s
[DEBUG] 18:41:33.123 shell.go:Host:560 → DESKTOP-OGJO8A0
[TRACE] 18:41:33.123 shell.go:Host() - 1.003ms
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.123 shell.go:Getenv:397 → NO DATA
[TRACE] 18:41:33.123 shell.go:Getenv(SHLVL) - 0s
[TRACE] 18:41:33.123 shell.go:TemplateCache() - 1.003ms
[TRACE] 18:41:33.123 shell.go:Flags() - 0s
[DEBUG] 18:41:33.123 block.go:Debug:280 → Segment: session
[TRACE] 18:41:33.123 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.123 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.123 shell.go:Getenv:397 → NO DATA
[TRACE] 18:41:33.123 shell.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 18:41:33.123 shell.go:Getenv:397 → NO DATA
[TRACE] 18:41:33.123 shell.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 18:41:33.123 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.123 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.123 shell.go:Shell() - 0s
[DEBUG] 18:41:33.123 block.go:Debug:280 → Segment: path
[TRACE] 18:41:33.123 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.123 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:Pwd() - 0s
[TRACE] 18:41:33.123 shell.go:Shell() - 0s
[TRACE] 18:41:33.123 shell.go:Flags() - 0s
[DEBUG] 18:41:33.123 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 18:41:33.123 properties.go:GetString:28 → 
[DEBUG] 18:41:33.123 properties.go:GetString:28 → 
[DEBUG] 18:41:33.123 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.123 properties.go:GetString:28 → ~
[DEBUG] 18:41:33.123 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 18:41:33.123 shell.go:PathSeparator() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:PathSeparator() - 0s
[TRACE] 18:41:33.123 shell.go:PathSeparator() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:PathSeparator() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:PathSeparator() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:PathSeparator() - 0s
[DEBUG] 18:41:33.123 properties.go:GetString:28 → folder
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:GOOS() - 0s
[TRACE] 18:41:33.123 shell.go:PathSeparator() - 0s
[DEBUG] 18:41:33.123 properties.go:GetString:28 → NO DATA
[TRACE] 18:41:33.123 shell.go:PathSeparator() - 0s
[DEBUG] 18:41:33.123 properties.go:GetString:28 → \
[TRACE] 18:41:33.123 shell.go:Pwd() - 0s
[TRACE] 18:41:33.123 shell_windows.go:IsWsl() - 0s
[TRACE] 18:41:33.123 shell.go:StackCount() - 0s
[DEBUG] 18:41:33.124 win32_windows.go:isWriteable:303 → current user is member of S-1-5-21-4000869473-2540451011-1271548399-1000
[DEBUG] 18:41:33.124 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 18:41:33.124 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 18:41:33.124 shell_windows.go:DirIsWritable() - 1.0002ms
[TRACE] 18:41:33.124 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.124 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.124 shell.go:Shell() - 0s
[DEBUG] 18:41:33.124 block.go:Debug:280 → Segment: git
[TRACE] 18:41:33.124 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.124 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.124 shell.go:GOOS() - 0s
[TRACE] 18:41:33.125 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.141 shell.go:CommandPath:599 → C:\Program Files\Git\cmd\git.exe
[TRACE] 18:41:33.141 shell.go:CommandPath(git.exe) - 15.9988ms
[TRACE] 18:41:33.141 shell.go:HasCommand(git.exe) - 15.9988ms
[TRACE] 18:41:33.141 shell.go:Pwd() - 0s
[ERROR] 18:41:33.141 shell.go:HasParentFilePath:732 → CreateFile .git: The system cannot find the file specified.
[TRACE] 18:41:33.141 shell.go:HasParentFilePath(.git) - 0s
[DEBUG] 18:41:33.141 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 18:41:33.141 block.go:Debug:280 → Segment: root
[TRACE] 18:41:33.141 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.141 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.141 shell.go:GOOS() - 0s
[TRACE] 18:41:33.141 shell_windows.go:Root() - 0s
[DEBUG] 18:41:33.142 block.go:Debug:280 → Segment: exit
[TRACE] 18:41:33.142 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.142 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.142 shell.go:GOOS() - 0s
[TRACE] 18:41:33.142 shell.go:ErrorCode() - 0s
[DEBUG] 18:41:33.142 properties.go:GetBool:22 → always_enabled: true
[TRACE] 18:41:33.142 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.142 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.142 shell.go:Shell() - 0s
[TRACE] 18:41:33.142 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.142 shell.go:Flags() - 0s
[DEBUG] 18:41:33.142 block.go:Debug:280 → Segment: node
[TRACE] 18:41:33.142 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.142 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.142 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.142 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .npmrc .vue]
[DEBUG] 18:41:33.142 properties.go:GetStringArray:52 → folders: []
[DEBUG] 18:41:33.142 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:41:33.142 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.142 shell_windows.go:56 → C:\Users\Herbert
[DEBUG] 18:41:33.142 properties.go:GetString:28 → files
[DEBUG] 18:41:33.142 properties.go:GetBool:22 → fetch_package_manager: false
[TRACE] 18:41:33.142 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.142 shell.go:HasFiles:446 → true
[TRACE] 18:41:33.142 shell.go:HasFiles(*.js) - 0s
[DEBUG] 18:41:33.142 properties.go:GetBool:22 → fetch_version: true
[DEBUG] 18:41:33.150 shell.go:CommandPath:599 → C:\Program Files\nodejs\node.exe
[TRACE] 18:41:33.150 shell.go:CommandPath(node) - 7.9985ms
[TRACE] 18:41:33.150 shell.go:HasCommand(node) - 7.9985ms
[DEBUG] 18:41:33.189 shell.go:RunCommand:578 → v18.16.1
[TRACE] 18:41:33.189 shell.go:RunCommand(node --version) - 39.0797ms
[DEBUG] 18:41:33.189 properties.go:GetString:28 → https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V{{ .Major }}.md#{{ .Full }}
[TRACE] 18:41:33.189 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.189 shell.go:Pwd() - 0s
[ERROR] 18:41:33.190 shell.go:FileContent:518 → open D:\Repos\Projects\expense-tracker\backend\.nvmrc: The system cannot find the file specified.
[TRACE] 18:41:33.190 shell.go:FileContent(.nvmrc) - 539.8µs
[TRACE] 18:41:33.190 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.190 shell.go:Shell() - 0s
[DEBUG] 18:41:33.190 block.go:Debug:280 → Segment: go
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.190 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.190 properties.go:GetStringArray:52 → extensions: [*.go go.mod]
[DEBUG] 18:41:33.190 properties.go:GetStringArray:52 → folders: []
[DEBUG] 18:41:33.190 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell_windows.go:56 → C:\Users\Herbert
[DEBUG] 18:41:33.190 properties.go:GetString:28 → files
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell.go:HasFiles:449 → false
[TRACE] 18:41:33.190 shell.go:HasFiles(*.go) - 0s
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell.go:HasFiles:449 → false
[TRACE] 18:41:33.190 shell.go:HasFiles(go.mod) - 0s
[DEBUG] 18:41:33.190 block.go:Debug:280 → Segment: python
[DEBUG] 18:41:33.190 properties.go:GetString:28 → files
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.190 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.190 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 18:41:33.190 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv env venv-win pyenv-win]
[DEBUG] 18:41:33.190 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell_windows.go:56 → C:\Users\Herbert
[DEBUG] 18:41:33.190 properties.go:GetBool:22 → fetch_virtual_env: false
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell.go:HasFiles:449 → false
[TRACE] 18:41:33.190 shell.go:HasFiles(*.py) - 0s
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell.go:HasFiles:449 → false
[TRACE] 18:41:33.190 shell.go:HasFiles(*.ipynb) - 0s
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell.go:HasFiles:449 → false
[TRACE] 18:41:33.190 shell.go:HasFiles(pyproject.toml) - 0s
[TRACE] 18:41:33.190 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.190 shell.go:HasFiles:449 → false
[TRACE] 18:41:33.190 shell.go:HasFiles(venv.bak) - 0s
[DEBUG] 18:41:33.190 shell.go:HasFolder:492 → false
[TRACE] 18:41:33.190 shell.go:HasFolder(.venv) - 0s
[DEBUG] 18:41:33.190 shell.go:HasFolder:492 → false
[TRACE] 18:41:33.190 shell.go:HasFolder(venv) - 0s
[DEBUG] 18:41:33.190 shell.go:HasFolder:492 → false
[TRACE] 18:41:33.190 shell.go:HasFolder(virtualenv) - 0s
[DEBUG] 18:41:33.190 shell.go:HasFolder:492 → false
[TRACE] 18:41:33.190 shell.go:HasFolder(env) - 0s
[DEBUG] 18:41:33.191 shell.go:HasFolder:492 → false
[TRACE] 18:41:33.191 shell.go:HasFolder(venv-win) - 507.3µs
[DEBUG] 18:41:33.191 shell.go:HasFolder:492 → false
[TRACE] 18:41:33.191 shell.go:HasFolder(pyenv-win) - 0s
[DEBUG] 18:41:33.191 block.go:Debug:280 → Segment: shell
[TRACE] 18:41:33.191 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.191 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.191 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.191 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 18:41:33.191 shell.go:Shell() - 0s
[TRACE] 18:41:33.191 shell.go:Flags() - 0s
[TRACE] 18:41:33.191 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.191 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.191 shell.go:Shell() - 0s
[DEBUG] 18:41:33.191 block.go:Debug:280 → Segment: time
[TRACE] 18:41:33.191 shell.go:Pwd() - 0s
[DEBUG] 18:41:33.191 shell_windows.go:56 → C:\Users\Herbert
[TRACE] 18:41:33.191 shell.go:GOOS() - 0s
[DEBUG] 18:41:33.191 properties.go:GetString:28 → 15:04:05
[TRACE] 18:41:33.191 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.191 shell.go:TemplateCache() - 0s
[TRACE] 18:41:33.191 shell.go:Shell() - 0s
[DEBUG] 18:41:33.191 shell.go:Getenv:397 → C:\Users\Herbert\AppData\Local
[TRACE] 18:41:33.191 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 18:41:33.191 shell_windows.go:CachePath() - 57.7µs
[TRACE] 18:41:33.191 shell.go:Flags() - 0s

**Reply by walherb** [reply]
@JanDeDobbeleer any feedback about the above problem

**Reply by JanDeDobbeleer** [reply]
@walherb I went through the logs and didn't see anything strange. The same logs on my end display the version, I yet have to check the default configuration to see if that changes anything but didn't get around to that yet.

**Reply by JanDeDobbeleer** [reply]
@walherb it's because the default config [doesn't have the version in its template](https://github.com/JanDeDobbeleer/oh-my-posh/blob/0570e5054986893bd4f8a3e05d4d554fbc475f72/src/engine/config.go#L424). You can [export the configuration](https://ohmyposh.dev/docs/installation/customize#adjust-a-theme) and add it as per the [docs](https://ohmyposh.dev/docs/segments/node).

**Reply by walherb** [reply]
Am also getting perplexed because it was working well with version 17.4.0, if this could help am on Windows 10 Enterprise LTSC 21H2 19044.3086 and the theme for oh-my-posh is jandedobbeleer.omp.json

**Reply by JanDeDobbeleer** [reply]
@walherb the template probably got changed, I would always suggest to create your own configuration as it allows to easily tweak these things out of the box.

**Reply by walherb** [reply]
Yes, the template got changed, I had to download the theme of version 17.4.0 because I recall that one everything worked well so I pointed my theme config to the one of 17.4.0 and everything is now ok.
